.github/workflows/docker.yaml

```diff
@@ -10,7 +10,7 @@ on:
       - main
 
 env:
-  SUI_GIT_REVISION: testnet-v1.35.0
+  SUI_GIT_REVISION: ad41ab4   # testnet release with WS filter hotfix
```

Comment:
With the recent change to how we clone the repo, you may need to use the full commit `ad41ab49e72862cb74721b2100321cf0695eb5d0`